frontend/controller/actions/group.js

```diff
@@ -995,6 +995,29 @@ export default (sbp('sbp/selectors/register', {
 
     return sendMessage(params)
   }),
+  'gi.actions/group/updateLastLoggedIn': async ({ contractID }: { contractID: string }) => {
+    try {
+      const rootState = sbp('state/vuex/state')
+      const userID = rootState.loggedIn?.identityContractID
+
+      if (!userID) {
+        throw new Error('Unable to update last logged in without an active session')
+      }
+      const now = new Date().toISOString()
+
+      // Wait for any pending operations (e.g., sync) to finish
+      await sbp('chelonia/queueInvocation', contractID, async () => {
+        const current = await sbp('chelonia/kv/get', contractID, 'lastLoggedIn')?.data || {}
+        current[userID] = now
+        await sbp('chelonia/kv/set', contractID, 'lastLoggedIn', current, {
+          encryptionKeyId: sbp('chelonia/contract/currentKeyIdByName', contractID, 'cek'),
+          signingKeyId: sbp('chelonia/contract/currentKeyIdByName', contractID, 'csk')
+        })
```

Comment:
Missing `catch`

Reply:
Also doesn't have `catch` on purpose, as an error here means updating the last logged in info (the caller's intent) failed.